History.md

```diff
@@ -1,162 +1,237 @@
-v3.4.1-beta.1 / 2019-06-20
```

Comment:
Some versions (tags) are missing. As a result, the changelog for some of the versions is not accurate.